ci-operator/config/openshift/metallb/openshift-metallb-release-4.9.yaml

```diff
@@ -49,25 +49,31 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws
-- as: metallb-l2-e2e-metal-ipi
+- as: metallb-e2e-metal-ipi
   steps:
     cluster_profile: packet
     env:
       DEVSCRIPTS_CONFIG: |
         IP_STACK=v4
         NETWORK_TYPE=OpenShiftSDN
-        ENABLE_METALLB_MODE=l2
+        ENABLE_METALLB=true
         METALLB_REPO=https://github.com/openshift/metallb.git
+        METALLB_OPERATOR_REPO=https://github.com/openshift/metallb-operator.git
+        METALLB_OPERATOR_COMMIT=22afa1a
```

Comment:
Same as above. This is even worse for the time being, no? It means we would be installing a newer version than the one shipped in OCP 4.9.

Reply:
here im still wondering if we are going to backport the FRR stuff to 4.9 both in metallb and in the operator

Reply:
We will not.

Reply:
so i will set it to a 4.9 commit

Reply:
I was more suggesting setting it to "release-4.9" than a specific commit.

Reply:
i know, let's decide first if we want a branch or commit.